src/main/java/org/apache/flink/graph/streaming/WindowGraphAggregation.java

```diff
@@ -1,9 +1,7 @@
 package org.apache.flink.graph.streaming;
 
-import org.apache.flink.api.common.functions.FoldFunction;
-import org.apache.flink.api.common.functions.MapFunction;
-import org.apache.flink.api.common.functions.ReduceFunction;
-import org.apache.flink.api.common.functions.RichMapFunction;
+import org.apache.flink.api.common.functions.*;
```

Comment:
can you please remove the wildcard import and import the required functions separately?